CHANGELOG/CHANGELOG-1.12.md

```diff
@@ -0,0 +1,55 @@
+# Changelog 1.12
+
+## 1.12.0 - 2023-05-24
+
+- The Multicast, TopologyAwareHints, NodeIPAM features are graduated from Alpha to Beta and therefore enabled by default.
+
+### Added
+
+- Add a new field 'logLabel' to ACNP and ANP CRDs to specify firewall log label for Audit Logging. ([#4748](https://github.com/antrea-io/antrea/pull/4748), [@qiyueyao])
+- Add support for short-circuiting in AntreaProxy to ensure that the traffic from Pod/Node clients to
+external addresses behaves the same way as the traffic from external clients to external addresses. ([#4815](https://github.com/antrea-io/antrea/pull/4815), [@hongliangl])
+- Add support for ExternalIP in AntreaProxy to allows a Service to be accessed from outside the cluster using a static IP address. ([#4866](https://github.com/antrea-io/antrea/pull/4866), [@hongliangl])
+- Add two new fields 'sourcePort' and 'sourceEndPort' in Antrea-native policy API to match traffic initiated from specific ports. ([#4687](https://github.com/antrea-io/antrea/pull/4687), [@Dyanngg])
+- Add OVS table name as label for 'ovs_flow_count' Prometheus metrics. ([#4893](https://github.com/antrea-io/antrea/pull/4893), [@cr7258])
+- Add WireGuard tunnel mode for Multi-cluster to establish encrypted connection between member clusters. ([#4606](https://github.com/antrea-io/antrea/pull/4606), [@hjiajing])
+- Watch EndpointSlice API by default for exported Service's endpoint change in Multi-cluster. ([#4895](https://github.com/antrea-io/antrea/pull/4895), [@luolanzone])
```

Comment:
I feel there is no need to highlight PodIP Endpoints at first sentence. MC also watches EndpointSlice API to detect if there is any available Endpoints for exported Service when it's ClusterIP type, and may remove the corresponding ResourceExport if there is no available Endpoints. Maybe we can say:

"Add support for EndpointSlice API for Multi-cluster Services. When the EndpointSlice API is available for the cluster, EndpointSlice resources of the exported Service, rather than the Endpoints resource, will be watched. For the Pod IP type Endpoints, EndpointSlice resources will be exported as the Multi-cluster Service's endpoints."

Reply:
The thing is "watch" is more internal implementation and not very interesting to users. How about: "watched" -> "processed"

Then maybe remove "For the Pod IP type of Services, EndpointSlice resources will be exported as the Multi-cluster Service's endpoints." First, there is no "Pod IP type of Services" and no one will understand what does it mean. Second, finally we still export and create Endpoints, not EndpointSlice, and the sentence may be misleading to say "EndpointSlice resources will be exported".

Reply:
Updated.